### DISCUSSION TITLE
SCTransform V2 integrated data analysis reveals no DE genes in clusters

### DISCUSSION THREAD

**Discussion by Prathyusha-konda** [satijalab/seurat#6574] [Q&A]
Hi, I performed integrated analysis using SCT V2, which identified 15 clusters, but when I perform DE analysis using FindAllMarkers function on data slot with default options except for pct=0.1 cutoff, there are no DE genes identified in one of the clusters, which is a rather significant cluster with 10,000+ cells. I am trying to interpret what this means - why would these cells be clustered separately if there are no DE genes?

(Note: when I do FindAllMarkers on scale.data slot, it identified DE genes in all clusters, but based on all the discussions here, I understand this is not appropriate to be done on scale.data)

Would appreciate any insights here. Thanks in advance.

**Comment by munghong** [comment]
It might be an issue of the newest Seurat update version 4.2?  I also got few DE genes using FindMarkers for my integrated analysis. So I switched back to version 4.1 and got many many more DE genes.
